content/applications/hr/fleet/models.rst

```diff
@@ -0,0 +1,194 @@
+| A1    | Serie 1      | Class A   | Agilia       |
++-------+--------------+-----------+--------------+
+| A3    | Serie 3      | Class B   | Ampera       |
++-------+--------------+-----------+--------------+
+| A4    | Serie 5      | Class C   | Antara       |
++-------+--------------+-----------+--------------+
+| A5    | Serie 6      |Class CL   | Astra        |
++-------+--------------+-----------+--------------+
+| A6    | Serie 7      | Class CLS | AstraGTC     |
++-------+--------------+-----------+--------------+
+| A7    | Serie Hybrid | Class E   | Combo Tour   |
++-------+--------------+-----------+--------------+
+| A8    | Serie M      | Class GL  | Corsa        |
++-------+--------------+-----------+--------------+
+| Q3    | Serie X      | Class GLK | Insignia     |
++-------+--------------+-----------+--------------+
+| Q5    | Serie Z4     | Class M   | Meriva       |
++-------+--------------+-----------+--------------+
+| Q7    |              | Class R   | Mokka        |
++-------+--------------+-----------+--------------+
+| TT    |              | Class S   | Zafira       |
++-------+--------------+-----------+--------------+
+|       |              | Class SLK | Zafira Tourer|
++-------+--------------+-----------+--------------+
+|       |              | Class SLS |              |
++-------+--------------+-----------+--------------+
+
+.. _fleet/add-model:
+
+Add a new model
+===============
+
+New vehicle models can easily be added to the database. To add a new model, navigate to
+:menuselection:`Fleet app --> Configuration --> Models: Models`. Click :guilabel:`New` and a
+vehicle model form loads. Enter the following information on the form. Be advised, some fields are
+specific to Belgian based companies, so not all fields or sections may be visible depending on the
+location of the company.
+
+.. image:: models/mini.png
+   :align: center
+   :alt: A new model form with the informatoin filled out for a Mini Cooper.
+
+- :guilabel:`Model name`: enter the model name in the field.
+- :guilabel:`Manufacturer`: select the manufacturer from the drop-down menu. If the manufacturer is
+  not configured, type in the manufacturer and then click :guilabel:`Create` or :guilabel:`Create
+  and edit..`.
+- :guilabel:`Vehicle Type`: select one of two preconfigured vehicle types, either :guilabel:`Car` or
+  :guilabel:`Bike`, from the drop-down menu. The vehicle types are hardcoded in Odoo and are
+  integrated with the *Payroll* application since vehicles can be part of an employee's benefits.
+  Adding additional vehicle types is not possible as it affects payroll.
+- :guilabel:`Category`: select the category the vehicle is categorized under from the drop-down
+  menu. To create a new category, type in the category and then click :guilabel:`Create (new
+  category)`.
+
+.. note::
+   When the manufacturer is selected, the logo for the manufacturer will automatically load in the
+   image box in the top-right corner.
```

Comment:
Since this relates to the Manufacturer field, how about moving this note block up to line 68?